app/controllers/miq_task_controller.rb

```diff
@@ -445,19 +447,17 @@ def get_layout
   end
 
   def get_session_data
-    @layout        = get_layout
-    @jobs_tab      = session[:jobs_tab] if session[:jobs_tab]
+    super
+    @layout = get_layout
+    @tasks_tab     = session[:tasks_tab] if session[:tasks_tab]
```

Comment:
Good idea to rename `jobs_tab` for consistency, *but*...

```
$ ag @jobs_tab
app/controllers/miq_task_controller.rb
449:    @jobs_tab      = session[:jobs_tab] if session[:jobs_tab]
456:    session[:jobs_tab]            = @jobs_tab
```

Meaning we only ever save it to session and restore it from session.

Aand since we never actually use it, you can just drop it :)